src/common/pokemonStorage.js

```diff
@@ -1,23 +1,85 @@
 const STORAGE_KEY = "cart";
+const COUNTER_KEY = "counter";
+const ORDER_KEY = "order";
+
+export const getOrderDetails = () => {
+  try {
+    const orderStorage = localStorage.getItem(ORDER_KEY);
+    return orderStorage ? JSON.parse(orderStorage) : [];
+  } catch (e) {
+    return [];
+  }
+};
 
 export const getCartItems = () => {
   try {
     const cartStorage = localStorage.getItem(STORAGE_KEY);
-    return JSON.parse(cartStorage);
+    return cartStorage ? JSON.parse(cartStorage) : [];
+  } catch (e) {
+    return [];
+  }
+};
+
+export const getCounter = () => {
+  try {
+    const counterStorage = localStorage.getItem(COUNTER_KEY);
+    if (counterStorage) {
+      /* eslint-disable */
+      const count = parseInt(counterStorage);
+      return count;
+    }
+    return 0;
   } catch (e) {
     return [];
   }
 };
 
 export const addToCart = (item) => {
   const cartStorage = getCartItems();
+  const currentCounter = getCounter();
+  const foundIndex = cartStorage.findIndex((itm) => itm.name === item.name);
```

Comment:
same here about names: `itm` is not really a good name. In such case I'll go with just `i` as a short from item, or with full `cartItem` which probably better because you see instantly that you're trying to find same nae in cart